content/en/Getting started/aws.md

```diff
@@ -27,6 +27,10 @@ This set up the base infrastructure (like network and cluster) that is the found
 > Note that it costs around $5 per day to run this on AWS. So make sure to destroy it after you're done 
 > (opta has a destroy command so it should be easy :))!
 
+> Note that Opta requires you to use a region with at least 3 availability zones. This is not only
```

Comment:
I understand the user wants this in getting started, but given how clear our error message is in the cli and the rarity that this would happen. I don't think this is needed here.